cmds/cpud/serve.go

```diff
@@ -115,6 +118,31 @@ func serve() error {
 	if err != nil {
 		return err
 	}
+	// TODO: Move to server package
+	if *mdnsEnabled {
+		v("starting mdns server")
+		// Setup our service export
+		host, _ := os.Hostname()
+		var p uint64
+		p, err = strconv.ParseUint(*port, 10, 32)
```

Comment:
Are ports 32 bits? Do we need to require base 10? People might want to say port 0xcafe maybe?

Reply:
Agreed.  will stop specifying the port conversion.